gegelatilib/src/learn/learningAgent.cpp

```diff
@@ -396,6 +399,20 @@ Learn::LearningAgent::getBestRoot() const
     return this->bestRoot;
 }
 
+void Learn::LearningAgent::updateBestScoreLastGen(
+    std::multimap<std::shared_ptr<Learn::EvaluationResult>,
+                  const TPG::TPGVertex*>& results)
+{
+    auto iter = results.begin();
+    std::advance(iter, results.size() - 1);
```

Comment:
I think there could be a faster code to retrieve the last element of the map. Like in line 377, retrieving the `--results.end()` iterator could do the trick in just one hop.